src/Admin/CkeditorAdminExtension.php

```diff
@@ -30,11 +30,11 @@ final class CkeditorAdminExtension extends AbstractAdminExtension
     public function configureRoutes(AdminInterface $admin, RouteCollectionInterface $collection): void
     {
         $collection->add('ckeditor_browser', 'ckeditor_browser', [
-            '_controller' => CkeditorAdminController::class.'::browserAction',
+            '_controller' => 'sonata.formatter.ckeditor.extension.controller::browserAction',
         ]);
 
         $collection->add('ckeditor_upload', 'ckeditor_upload', [
-            '_controller' => CkeditorAdminController::class.'::uploadAction',
+            '_controller' => 'sonata.formatter.ckeditor.extension.controller::uploadAction',
```

Comment:
Why this change ?

Reply:
When you call controller from route this way:
CkeditorAdminController::class.'::uploadAction',
 then controller class can be loaded via PSR4 rather than via service name in the container. As a result the container is not loaded into the controller (because relevant initialization hooks were not called) and it fails.
 
Basically - to make sure controller is loaded as service via container.